models/notifications/INotification.ts

```diff
@@ -0,0 +1,13 @@
+import { NotificationType } from './NotificationType';
+import IUser from "./../users/IUser"
+
+/**
+ * Model interface for a notification.
+ */
+export default interface IConversion {
+  type?: NotificationType;
+  notificationString: string; 
+  userNotified: IUser;
```

Comment:
What's the difference between userNotified and acting? Is user acting the one who triggered the notification and the userNotified the who is receiving the notification?